src/app/modules/user/selectors.ts

```diff
@@ -0,0 +1,12 @@
+import { RootState } from '../../store';
+import api from '../../api/index';
+import Client from '../../api/Client';
+
+export const currentCourthouse = (state: RootState): string => {
+    const { user: { currentCourthouse: courthouse = '' } = {} } = state;
+    return courthouse;
+};
+
+export const isCourthouseSet = (): boolean => {
+    return (api as Client).isCourthouseSet;
```

Comment:
instead of pulling from the api client here, we should be pulling from state.  Since we want our state to be the source of truth:

```ts
export const isCourthouseSet = createSelector(
    currentCourthouse,
    (courthouse) => {
        // tslint:disable-next-line:triple-equals
        return courthouse != undefined && courthouse !== '';
    }
);
```